.circleci/DSDTI_outputs.txt

```diff
@@ -6,15 +6,15 @@ qsiprep/logs/CITATION.md
 qsiprep/logs/CITATION.tex
 qsiprep/sub-PNC
 qsiprep/sub-PNC/anat
-qsiprep/sub-PNC/anat/sub-PNC_desc-brain_mask.nii.gz
-qsiprep/sub-PNC/anat/sub-PNC_desc-preproc_T1w.nii.gz
-qsiprep/sub-PNC/anat/sub-PNC_dseg.nii.gz
+qsiprep/sub-PNC/anat/sub-PNC_space-ACPC_desc-brain_mask.nii.gz
+qsiprep/sub-PNC/anat/sub-PNC_space-ACPC_desc-preproc_T1w.nii.gz
+qsiprep/sub-PNC/anat/sub-PNC_space-ACPC_dseg.nii.gz
 qsiprep/sub-PNC/anat/sub-PNC_from-MNI152NLin2009cAsym_to-T1w_mode-image_xfm.h5
 qsiprep/sub-PNC/anat/sub-PNC_from-orig_to-T1w_mode-image_xfm.txt
 qsiprep/sub-PNC/anat/sub-PNC_from-T1w_to-MNI152NLin2009cAsym_mode-image_xfm.h5
```

Comment:
Should this be `from-ACPC_to-MNI`?

Reply:
Ah right, I didn't consistently update the files in `.circleci`. I'm not actually sure which ones are still being used. A bunch of them are duplicates of the tests in `test_cli.py`, right?

Reply:
🤦 I didn't see the directory this is in. We should remove most of this directory at some point, since it doesn't work anymore